pkg/server/spec/pod.go

```diff
@@ -55,6 +57,12 @@ func newPodContainers(name, nginxImage, appImage string, envVars map[string]stri
 		newAppContainer(name, appImage, envVars, appPort, secrets),
 	}
 	if nginxImage != "" {
+		c[0].VolumeMounts = []*VolumeMounts{
```

Comment:
Mixed feelings about that, maybe this can be moved to `newAppContainer`.

Reply:
`newAppContainer` func doesn't know about nginx right?

Reply:
`newAppContainer` receive `secondaryPort` as `port` value, but I'm not sure if it's a good approach

Reply:
hm, I'm lost with tons of doubts LOL, @aguerra what do you think about it?

Reply:
:shipit: